src/code/actions.js

```diff
@@ -268,12 +271,42 @@ export function submitDrake(correctPhenotype, submittedPhenotype, correct, incor
   };
 }
 
-function _putEggInBasket(egg, basket, isCorrect) {
+function _rejectEggFromBasket(eggIndex, basketIndex) {
+  return {
+    type: actionTypes.EGG_REJECTED,
+    eggIndex,
+    basketIndex
+  };
+}
+
+export function rejectEggFromBasket() {
+  return (dispatch, getState) => {
+    const { submittedEggIndex, submittedBasketIndex } = getState();
+    dispatch(_rejectEggFromBasket(submittedEggIndex, submittedBasketIndex));
+  };
+}
+
+function _acceptEggInBasket(eggIndex, basketIndex) {
+  return {
+    type: actionTypes.EGG_ACCEPTED,
+    eggIndex,
+    basketIndex
+  };
+}
+
+export function acceptEggInBasket() {
+  return (dispatch, getState) => {
+    const { submittedEggIndex, submittedBasketIndex } = getState();
+    dispatch(_acceptEggInBasket(submittedEggIndex, submittedBasketIndex));
+  };
+}
+
+function _submitEggForBasket(eggIndex, basketIndex, isCorrect) {
```

Comment:
Suggestion #2 would have to deal with the issue that there can be multiple eggs/drakes with the same genotype/phenotype, so I went with #1 and renamed the field to `eggDrakeIndex`.